CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set_property(GLOBAL PROPERTY JOB_POOL_LINK local_jobs)
 
 # First include general CMake utilities.
 include(SwiftUtils)
+include(CheckSymbolExists)
```

Comment:
Nitpick: this belongs down with the other `include(Check*)` lines (where you put the new one).